test/unit/sourceset.test.js

```diff
@@ -443,6 +715,7 @@ QUnit[qunitFn]('sourceset', function(hooks) {
     });
 
     QUnit.test('mediaEl.load()', function(assert) {
+      const done = assert.async();
```

Comment:
this shouldn't be necessary since everything happens synchronously.

Reply:
agreed